WireAPI/Sources/WireAPI/APIs/ConversationsAPI/ConversationsAPIV0.swift

```diff
@@ -170,45 +170,45 @@ struct ConversationV0: Decodable, ToAPIModelConvertible {
         case type
     }
 
-    var access: [String]?
-    var accessRoles: [String]?
+    var access: Set<ConversationAccessMode>?
+    var accessRoles: Set<ConversationAccessRole>?
     var creator: UUID?
     var epoch: UInt?
     var id: UUID?
     var lastEvent: String?
-    var lastEventTime: String?
-    var legacyAccessRole: String?
+    var lastEventTime: Date?
```

Comment:
I had discovered an issue regarding timestamps a while ago.
Sometimes the backend sends timestamps with fractional seconds, sometimes without.
I wasn't able to have a single parser supporting both, so I added a workaround.
The ticket `WPB-6529` is mentioned in some TODO the code.
Please look at the comments in JIRA:
https://wearezeta.atlassian.net/browse/WPB-6529

Reply:
hmm so it looks like backend just updated the documentation and change the format to be consistent. I'll take a second look and make sure we're decoding with the correct format. Thanks

Reply:
So it looks like that are payloads that have both time formats: for instance the `Client` payload has a `last_active` field that has time without milliseconds, and `time` with milliseconds. To handle this I can see two options:

1. Do manual decoding to handle decode the fields separately with different date decoders, or
2. Bring back the workaround that tries to decode with milliseconds then falls back to without milliseconds.

Option 2 seems fine to me, what do you think?

Reply:
Option 2 sounds good to me.

Reply:
done 2143688ee4898b1a141123ff33edd625261a8376